### ISSUE TITLE
Support for (binary) file download

### ISSUE THREAD

**Issue by qqilihq** [cellular/oazapfts#49]
Our API (fyi @danielesser) has an endpoint which produces a binary file meant for downloading. It looks like this:

```json
"/file/{fileId}/download": {
  "get": {
    "operationId": "DownloadFile",
    "responses": {
      "200": {
        "description": "Ok",
        "content": {
          "application/octet-stream": {
            "schema": {
              "type": "string",
              "format": "binary"
            }
          }
        }
      }
    },
    "description": "Get the content of a file.",
    "parameters": [
      {
        "in": "path",
        "name": "fileId",
        "required": true,
        "schema": {
          "$ref": "#/components/schemas/ObjectID"
        }
      }
    ]
  }
}
```

Currently, oazapfts would use the `oazapfts.fetchText` function to handle this, however a `Blob` would be more adequate (as in fact already used as parameter type for multipart file **upload**).

Would you potentially accept a MR adding such a functionality?

**Comment by qqilihq** [comment]
Pinging @fgnass (supposedly you're the creator/maintainer) about this again -- we have the mentioned functionality in a fork (see [here](https://github.com/cellular/oazapfts/compare/master...qqilihq:master)) which we’re already using successfully in two of our projects. I’d be very happy about a feedback:

Is this interesting for you to be merged back into the main project?

Many thanks in advance!

**Comment by fgnass** [comment]
Landed in oazapfts@3.2.0
